meta-lxatac-software/conf/distro/tacos.conf

```diff
@@ -0,0 +1,50 @@
+DISTRO = "tacos"
+DISTRO_NAME = "TAC OS - The LXA TAC operating system"
+DISTRO_VERSION = "23.11+dev"
+DISTRO_CODENAME = "tacos-mickledore"
```

Comment:
The convention for `DISTRO_VERSION` should be to set it to the anticipated release version once we start feature freezing and to `+dev` once we released and start to add features again.

The convention for `DISTRO_CODENAME` should be to update it whenever we change yocto releases.

Reply:
Perhaps add a comment here to avoid mistakes in the future?